dockers/base-conda/Dockerfile

```
@@ -12,13 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG CUDA_VERSION=11.1
+ARG CUDA_VERSION=11.3.1
 
-FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu18.04
+FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu20.04
```

Shall we keep Ubuntu18.04 for now because of the following error?
```
definition: docker.io/nvidia/cuda:10.2-devel-ubuntu20.04: not found
```
```suggestion
FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu18.04
```